openedx/adg/lms/webinars/models.py

```diff
@@ -438,6 +441,68 @@ def get_registered_and_unregistered_team_members(self):
 
         return team_emails_having_platform_accounts, team_emails_having_no_platform_accounts
 
+    def get_team_recipients_for_invitation_emails(self):
+        """
+        Get webinar team member emails (Panelists + Alternative hosts) to which webinar invitations will be sent.
+
+        1. Send invites to all team members when webinar is published for the first time.
+        2. Once a webinar is published then send invites to only new members.
+
+        Returns:
+            set: Contains emails of webinar team members.
+        """
+        old_webinar_data = getattr(self, '_loaded_values', {})
+        team_emails_for_invitations = self.team_emails
+
+        if old_webinar_data and old_webinar_data.get('is_published'):
+            old_panelists = extract_emails_from_string(old_webinar_data.get('panelists'))
+            old_alternative_hosts = extract_emails_from_string(old_webinar_data.get('alternative_hosts'))
+            old_team_members = set(old_panelists).union(set(old_alternative_hosts))
```

Comment:
let's do `set(old_panelists + old_alternative_hosts)` instead of calling set twice.